### ISSUE TITLE
Error on build MacOS

### ISSUE THREAD

**Issue by bateramos** [paulbartrum/jurassic#9]
Hello,

i'm trying to build jurassic on my MacOS, but I get some errors with the projects that they can't find the jurassic library. And I get this error also:

/Library/Frameworks/Mono.framework/Versions/4.2.1/lib/mono/4.5/Microsoft.Common.targets: Warning: Unable to find framework corresponding to the target framework moniker '.NETFramework,Version=v3.5,Profile=Client'. Framework assembly references will be resolved from the GAC, which might not be the intended behavior. (Jurassic)

and a lot of:

..../workspace/jurassic/Jurassic/CSC: Error CS0518: The predefined type `System.Object' is not defined or imported (CS0518) (Jurassic)

I really would love to contribute in this project. I love this idea and use this framework on my Unity3D projects.

**Comment by paulbartrum** [comment]
I don't have a mac, sorry, so I can't really help you.  One thing you could try: change the target framework in the Jurassic.csproj project file to `.NETFramework,Version=v3.5` (i.e. remove the Profile=Client).

**Comment by bateramos** [comment]
Thanks for the help.
I made the most of the project compile when change the target framework to .NET 4.0 but some projects can't compile because there are incompatible with Xamarin (Performance and Unit Tests). 
Also Xamarin nuget version is current below 3.0 and some dependencies need nuget version to be greater than that.
I also needed to update by hand the ICSharpCode.SharpZipLib, nuget was not getting the latest version, don't know why...
Later I will try to break all of the remaining issues and isolate them to see if we can solve this.
